src/items/external-blocks.md

```diff
@@ -211,6 +230,40 @@ extern {
 }
 ```
 
+Using this attribute with the `link_ordinal` attribute will result in a
+compiler error.
+
+### The `link_ordinal` attribute
+
+This attribute is only used with the `raw-dylib` linking kind.
+It is ignored if used with any other kind.
```

Comment:
@ehuss thoughts on this? (Should we raise an error if `link_ordinal` is used for other linking kinds?)

Reply:
It seems reasonable to make that an error.  The RFC seemed to imply that it can only be specified with raw-dylib.

Reply:
Ok, updated the docs and created a PR to change the behavior: https://github.com/rust-lang/rust/pull/100069